src/packages/dialog/types.ts

```
@@ -8,36 +9,39 @@ export type DialogConfigType = {
+  footerDirection: DialogFooterDirection
+  onClick: (event: MouseEvent<HTMLElement>) => void
+}
+export type DialogWrapProps = OverlayProps &
+  ContentProps & {
+    visible: boolean
+    overlay: boolean
+    overlayStyle: React.CSSProperties
+    overlayClassName: string
+    onCancel: () => void
+    onClose: () => void
+    onOverlayClick: (e: MouseEvent<HTMLElement>) => boolean | void
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)